docs/api-docs/apps/tutorials/sample-app-next/step-1-setup.mdx

```diff
@@ -77,14 +77,33 @@ npm install --save-dev babel-plugin-styled-components @types/node @types/react t
   "scripts": {
     "dev": "next",
     "build": "next build",
-    "start": "next start",
+    "start": "next start -p $PORT",
     "test": "echo \"Error: no test specified\" && exit 1"
   }
 }
 ```
 
 3. Save your changes.
 
+## Add the node and npm engines
```

Comment:
adding `engines` isn't a requirement of this app - this is more so to aid our development

Reply:
I'm going to leave this in for now to support the reader who's deploying this to a more complex environment.